build.gradle

```diff
@@ -41,12 +41,14 @@ subprojects {
     }
 
     dependencies {
+		
         compile "org.yaml:snakeyaml:latest.release"
+        
         compile "com.google.inject:guice:3.0"
         compile "com.google.inject.extensions:guice-servlet:3.0"
-        compile "com.sun.jersey.contribs:jersey-guice:1.9.1"
-        compile "com.sun.jersey:jersey-core:1.17.1"
-        compile "com.sun.jersey:jersey-server:1.17.1"
+        compile ("com.sun.jersey.contribs:jersey-guice:1.9.1")
```

Comment:
@ipapapa  actually there is no changes. I was excluding a library thats why you see the the () IF you want I can revert-it.